components/03-modules/header/_header.scss

```diff
@@ -1,32 +1,231 @@
-$header-padding   : 15px !default;
-$header-margin    : 15px !default;
-$header-background: $white !default;
+$header__text-border-color: $gray-dark !default;
+$header__border-color     : $gray !default;
+$header__font-size        : 12px !default;
+$header__text-decoration  : none !default;
 
 .header {
-    position: relative;
-    display: flex;
-    flex-wrap: nowrap;
-    justify-content: space-between;
-    padding: $header-padding 0;
-    background: $header-background;
-
-    &__right-panel {
+    border-bottom: 1px solid $header__border-color;
+
+    &__top-bar {
+        display: none;
+        background-color: $gray-light;
+        @include mq($screen-xl) {
+            display: block;
+        }
+
+        &-wrapper {
+            display: flex;
+            justify-content: space-between;
+            height: 30px;
+        }
+    }
+
+    &__contact-info {
+        margin: 8px 0;
+        font-size: $header__font-size;
+
+        &-link {
+            display: inline-flex;
+            margin-left: 8px;
+            padding-left: 8px;
+            border-left: 1px solid $header__text-border-color;
+            line-height: 14px;
+            text-decoration: $header__text-decoration;
+
+            &:focus,
+            &:hover {
+                text-decoration: $header__text-decoration;
+            }
+        }
+    }
+
+    &__links {
+        margin: 8px 0;
+        font-size: $header__font-size;
+
+        &-item {
+            padding: 0 16px;
+            border-left: 1px solid $header__text-border-color;
+            line-height: 14px;
```

Comment:
Use internal variables for this `px`values

Reply:
fixed